tests/helpers/common/Chocolatey/Invoke-Choco.ps1

```diff
@@ -18,8 +18,16 @@
     )
     begin {
         $chocoPath = Get-ChocoPath
-    $firstArgument, [string[]]$remainingArguments = $Arguments
-        $arguments = @($firstArgument; '--allow-unofficial'; $remainingArguments)
+        $firstArgument, [string[]]$remainingArguments = $Arguments
+        $arguments = @($firstArgument; '--allow-unofficial'; $remainingArguments
+            if ($env:CHOCO_TEST_PROXY) {
+                "--proxy='$($env:CHOCO_TEST_PROXY)'"
+                if ($env:CHOCO_TEST_PROXY_USER) {
+                    "--proxy-user='$($env:CHOCO_TEST_PROXY_USER)'"
+                    "--proxy-password='$($env:CHOCO_TEST_PROXY_PASSWORD)'"
+                }
+            }
+        )

```

Comment:
Since there are now a multiline argument, can you split all the arguments on multiple lines?

Also, as was found when adding the `--allow-unofficial` there could be problems when certain arguments are after other arguments, so it would be best if the `$remainingArguments` came last in this case.

```suggestion
        $arguments = @(
            $firstArgument
            '--allow-unofficial'

            if ($env:CHOCO_TEST_PROXY) {
                "--proxy='$($env:CHOCO_TEST_PROXY)'"
                if ($env:CHOCO_TEST_PROXY_USER) {
                    "--proxy-user='$($env:CHOCO_TEST_PROXY_USER)'"
                    "--proxy-password='$($env:CHOCO_TEST_PROXY_PASSWORD)'"
                }
            }

            $remainingArguments
        )

```